docs/api/changelog.rst

```diff
@@ -20,6 +20,13 @@ Fixed
   :function:`imod.select.points.point_values`
```

Comment:
Nice that you immediately update the changelog!
This should be part of our definition of done

Reply:
Thanks, I think our definition of done should be added to the developer docs, right?